node/modules/storageminer.go

```diff
@@ -351,6 +353,57 @@ func SectorStorage(mctx helpers.MetricsCtx, lc fx.Lifecycle, lstor *paths.Local,
 	return sst, nil
 }
 
+func F3Participation(mctx helpers.MetricsCtx, lc fx.Lifecycle, api v1api.FullNode, minerAddress dtypes.MinerAddress) error {
+	ctx := helpers.LifecycleCtx(mctx, lc)
+	b := &backoff.Backoff{
+		Min:    1 * time.Second,
+		Max:    1 * time.Minute,
+		Factor: 1.5,
+		Jitter: false,
+	}
+	go func() {
+		timer := time.NewTimer(0)
```

Comment:
Would it even if reset before first read?

Reply:
I don't believe it does, since `backoffWithContext` always resets the timer before selecting on it. But I'll leave the stop read dance be for safety

```go
func TestTimer_ZeroTimerWithResetDoesNotFireImmediately(t *testing.T) {
	timer := time.NewTimer(0)
	start := time.Now()
	timer.Reset(10 * time.Second)
	<-timer.C
	if time.Since(start) < 10*time.Second {
		t.Fail()
	}
}
```